src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
Okay, we're in agreement then :) I guess it's just something we need to document as a pitfall and folks will get used to it.

This change would have an impact on other usages of Dropdown though. I know the parameterized run stuff is using it, and it might be in the editor as well. Perhaps @scherler and @kzantow could chime in here? I definitely don't want to regress any other parts of the UI this late in the game.